### PR TITLE
Get StatsD and Sentry settings from env

### PR DESCRIPTION
**Pull request by meloie**
Signed-off-by: Keyhan Asadi <keyhan.asadi@cafebazaar.ir> (@keyhana)

### CODE REVIEW

File: snuba/settings_docker.py [hunk 1]
```diff
@@ -13,5 +13,7 @@
 USE_REDIS_CLUSTER = False
 
 # Dogstatsd Options
-DOGSTATSD_HOST = None
-DOGSTATSD_PORT = None
+DOGSTATSD_HOST = os.getenv("DOGSTATSD_HOST")
+DOGSTATSD_PORT = os.getenv("DOGSTATSD_PORT")
+
+SENTRY_DSN = os.getenv("SENTRY_DSN")
```
Comment (fpacifici) [inline snuba/settings_docker.py:19]: Any reason for not using the os.environ.get as in the lines above? Keeping these consistent would help maintainability as nobody would need to ask why some settings are loaded with os.getenv and others are not.

**Review by fpacifici — COMMENTED** [review]
Thanks, just one clarification.

File: snuba/settings_docker.py (hunk 1 above)
Comment (meloie) [reply snuba/settings_docker.py]: No actual reason and changed to be consistent with other ones. TnX

**Review by fpacifici — APPROVED** [review]

**Comment by keyhana** [comment]
@fpacifici Is there anything we can do to get this PR merged?

**Comment by evanh** [comment]
@keyhana This is good to be merged, we just can't update your branch with master manually right now. If you can update it again I'll merge it.

**Comment by keyhana** [comment]
@evanh Done.

**Pull request merged**